soapui/src/main/java/com/eviware/soapui/impl/rest/actions/explorer/EndpointExplorerAction.java

```diff
@@ -57,6 +60,7 @@ public void windowClosing(WindowEvent e) {
         String path = "/com/eviware/soapui/explorer/soapui-pro-api-endpoint-explorer-starter-page.html";
         String resource = this.getClass().getResource(path).toString();
         browser.navigate(resource);
+        browser.getPagePropertyMapper().update("dontShow", !SoapUI.getSettings().getBoolean(SHOW_ENDPOINT_EXPLORER_ON_START, true));
```

Comment:
Add null check as DisabledWebViewBasedBrowserComponent::getPagePropertyMapper() returns null. Or maybe you don't need this implementation as there is a default one in the interface.